src/js/components/sharedComponents/header/InfoBanner.jsx

```diff
@@ -26,37 +26,34 @@ export default class InfoBanner extends React.Component {
         return (
             <div className="info-banner">
                 <div className="info-banner__content">
-                    <div className="info-banner__alert-icon">
-                        <i className="usa-da-icon">
-                            <InfoCircle alt="Information" />
-                        </i>
-                    </div>
+                    <span className="info-banner__info-circle">
+                        <FontAwesomeIcon size="lg" icon="info-circle" />
+                    </span>
                     <div className="info-banner__alert-text">
                         <div className="info-banner__title-text">
-                            Welcome to the new USAspending.gov!
+                            Updated with FY 2019 Data - Your Guide to America&#39;s Finances
                         </div>
-                        We will continue to improve the data quality and display on a rolling basis.
-                        Questions? Check out our&nbsp;
-                        <a
-                            href="#/about"
-                            onClick={clickedBannerLink.bind(null, '/about')}>
-                            About
-                        </a> page for important information on the data and authoritative sources
-                        or join the conversation on our&nbsp;
+                        Your Guide provides a snapshot of 2019 Fiscal Year revenue,
+                        spending, deficit, debt, along with data for download.
+                        <span className="info-banner__hyphen">-</span>
                         <a
-                            href="https://usaspending-help.zendesk.com/hc/en-us/community/topics"
+                            href="https://datalab.usaspending.gov/americas-finance-guide/"
                             target="_blank"
                             rel="noopener noreferrer"
-                            onClick={clickedBannerLink.bind(null, 'https://usaspending-help.zendesk.com/hc/en-us/community/topics')}>
-                            Community
-                        </a> page.
+                            onClick={clickedBannerLink.bind(null, 'https://datalab.usaspending.gov/americas-finance-guide/')}>
```

Comment:
lets define this in the constructor to avoid creating an anon func every render.

Reply:
Could also just be a const inside the render.